Cannot install

The instructions for building and installing doesn't work
```
~ go get github.com/go-jira/jira/cmd/jira
# github.com/go-jira/jira/jiracli
go/src/github.com/go-jira/jira/jiracli/usage.go:157:45: cannot use app (type *"gopkg.in/alecthomas/kingpin.v2".Application) as type *"github.com/coryb/kingpeon/vendor/gopkg.in/alecthomas/kingpin.v2".Application in argument to kingpeon.RegisterDynamicCommandsWithRunner
```

Running macOS Catalina 10.15.1 Beta

After trying with both suggested fix or reverting the mentionned commit, I'm still left with the same issue:
```
$ go version && make
go version go1.10.4 linux/amd64
go build -gcflags="-e" -v -ldflags "-w -s" -o '$HOME/go/src/github.com/go-jira/jira/jira' cmd/jira/main.go
github.com/go-jira/jira/jiracli
# github.com/go-jira/jira/jiracli
jiracli/usage.go:157:45: cannot use app (type *"gopkg.in/alecthomas/kingpin.v2".Application) as type *"github.com/coryb/kingpeon/vendor/gopkg.in/alecthomas/kingpin.v2".Application in argument to kingpeon.RegisterDynamicCommandsWithRunner
Makefile:24: recipe for target 'build' failed
make: *** [build] Error 2
```

--- a/README.md
+++ b/README.md
@@ -14,13 +14,13 @@ You can download one of the pre-built binaries for **go-jira** [here](https://gi
 
 ### Build
 
-You can build and install the official repository with [Go](https://golang.org/dl/):
+You can build and install the official repository with [Go](https://golang.org/dl/) (before running the below command, ensure you have `GO111MODULE=on` set in your environment):
 
 	go get github.com/go-jira/jira/cmd/jira
 
 This will checkout this repository into `$GOPATH/src/github.com/go-jira/jira/`, build, and install it.
 
-It should then be available in $GOPATH/bin/jira
+It should then be available in $GOPATH/bin/jira.
 
 ## Usage
 

